src/Helpers/Utils.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Godot\AssetLibrary\Helpers;
 
+use PDO;
```

Comment:
The bindValue() method call on line 97 makes use of an explicit parameter type like described [here](http://php.net/manual/de/pdostatement.bindvalue.php).
It's a bit of a doubled safety measure because I'm also casting the value to int, but I think it's a good habit to use explicit parameter types with bindValue().